server/middleware/__tests__/standardLine.spec.js

```diff
@@ -10,7 +10,7 @@ describe('when the standardLine middleware is called', () => {
 
     const headers = '__headers__';
     const file = { key: '__key__', originalname: '__originalname__' };
```

Comment:
Picky i know, but should original name be camelcase?

Reply:
should be, but I think this is inline with the service